webapp/lib/playground.dart

```diff
@@ -0,0 +1,201 @@
+/**
+ * The entry point of the application.
+ */
+library coda.playground;
```

Comment:
I don't think playground is quite right here. 'mainUI?'

Reply:
Is this a view? A view model? An application shell? I'm not quite sure what role it plays in the app other than the entry point? If it's just the entry point, some stuff should probably be moved out into other classes

Reply:
I think it's basically an entry point and a view model of the whole thing. Depending how the code evolves, I may decide that it would be best to split this into two.